plenum/test/helper.py

```diff
@@ -347,7 +358,7 @@ def buildCompletedTxnFromReply(request, reply: Reply) -> Dict:
 def sendMessage(nodes: TestNodeSet,
                 frm: NodeRef,
                 to: NodeRef,
-                msg: Optional[Tuple]=None):
+                msg: Optional[Tuple] = None):
```

Comment:
From PEP8: "Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value."